lxr/checkout-version.py

```diff
@@ -1,8 +1,8 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
@iarspider , do we have python3 on current cmslxr node?

Reply:
Yes